components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -640,6 +649,35 @@ public static String retrieveValidatedGraphqlSchemaFromArchive(String pathToArch
         }
     }
 
+    /**
+     * Retrieve graphql complexity information from the file and validate it with the schema
+     *
+     * @param pathToArchive Path to API archive
+     * @param schema        GraphQL schema
+     * @throws APIImportExportException If an error occurs while reading the file
+     */
+    private static GraphqlComplexityInfo retrieveGraphqlComplexityInfoFromArchive(String pathToArchive, String schema) {
+
```

Comment:
Shall we remove unnecessary newlines since the logic inside the function is not lengthy? Here and in line 667 as well?

Reply:
Fixed with 2adb0a8